### PR TITLE
Issue 16: Populate report-an-issue.md

### LINKED ISSUE #16: Populate report-an-issue.md
Populate the `report-an-issue.md` file with detailed on the proper way to open an issue for this project in GitHub.  At least initially, let's limit the scope to reporting an issue with the documentation.  Later on we'll expand it for reporting an issue with any code in the project.

### PR DESCRIPTION
**Pull request by clementkng**
Fixes #16 

I based the documentation partially on https://wiki.lfedge.org/display/SDO/Reporting+Issues.

Signed-off-by: Clement Ng <clementdng@gmail.com>

### CODE REVIEW

File: common-requests/report-an-issue.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+ - Report
+author: clementkng
 ---
 
-# Report an Issue
+## Report an Issue
 
+If you would like to report an issue, visit the Github [issues section](https://github.com/open-horizon/open-horizon.github.io/issues), click the New issue button, and then fill out issue details.
+
+Please fill out a short, descriptive title for the issue and use the bigger text box to describe details such as current behavior, desired behavior, and what browser version you're seeing the issue on.
```
Comment (joewxboy) [inline common-requests/report-an-issue.md:16]: `what browser version you're seeing the issue on`

this line presumes the environment is a web browser.  Maybe instead we generalize a bit and instead say something like:

`operating system and version, architecture (arm32, arm64, x86, power, z1, quantum), device, and any other operating parameters that will help us replicate the issue`

**Review by Rene-Ch1 — CHANGES_REQUESTED** [review]

File: common-requests/report-an-issue.md (hunk 1 above)
Comment (clementkng) [reply common-requests/report-an-issue.md]: Thanks for the feedback, done.

File: common-requests/report-an-issue.md [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+description: ""
+tags:
+ - Help
+ - Issue
+ - Report
+author: clementkng
 ---
 
-# Report an Issue
+## Report an Issue
```
[the remark this answers is not in the document]
Comment (clementkng) [reply common-requests/report-an-issue.md]: Thanks for the feedback, done.

File: common-requests/report-an-issue.md [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
 
+If you would like to report an issue, visit the Github [issues section](https://github.com/open-horizon/open-horizon.github.io/issues), click the New issue button, and then fill out issue details.
```
[the remark this answers is not in the document]
Comment (clementkng) [reply common-requests/report-an-issue.md]: Done.

File: common-requests/report-an-issue.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (clementkng) [reply common-requests/report-an-issue.md]: Done.

**Review by joewxboy — APPROVED** [review]

**Review by Rene-Ch1 — APPROVED** [review]

**Comment by joewxboy** [comment]
This closes issue 16

**Pull request merged**